ISSUE_TEMPLATE

```diff
@@ -1,3 +1,5 @@
+Please prefer filing issues through the in-built extension command `CSharp: Start authoring a new issue on GitHub`. Know more - https://github.com/OmniSharp/omnisharp-vscode/wiki/Reporting-Issues.
```

Comment:
How about instead:
```
<!-- To prefill this information:
1. Open Visual Studio Code
2. Bring up the command palette (press <kbd>F1</kbd>)
3. Type `csharp.ReportIssue`
-->
```

Reply:
BTW: I choose that language because it is similar to what VS Code uses in its template -- https://github.com/Microsoft/vscode/issues/new?template=bug_report.md

Reply:
Actually, when I try this steps, the report issue command doesn't show up for me in C# extension version 1.17.1. Any idea why?

Reply:
Never mind. Figured it out.

Reply:
+1